### DISCUSSION TITLE
Plugin Debugging

### DISCUSSION THREAD

**Discussion by WoodenNautilus** [Eugeny/tabby#8383] [Q&A]
Hello,

One user of my plugin is having issues I cannot reproduce.
My idea is to give him a special version of the plugin which logs using the Logger class.

I know we can visualize these logs in the DevTools/Console but my issue is I don't know how to retrieve the data in a text file (for example). Is there a menu or an option in the DevTool to export the logs into a text file ?

Also, if you have a better idea to debug this issue, any tip or idea is welcomed ! The electron app and web development is pretty far from my domain of expertise.

**Comment by Eugeny** [comment]
These messages get written into the tabby/log.txt file which is next to the config file (in APPDATA or Application Support)

**Reply by WoodenNautilus** [reply]
Thanks @Eugeny
